udata_front/theme/gouvfr/assets/js/components/AdminAddMember/AdminAddMemberModal.vue

```diff
@@ -0,0 +1,122 @@
+<template>
+  <Teleport to="body">
+    <dialog
+      :aria-labelledby="modalTitleId"
+      role="dialog"
+      :id="id"
+      class="fr-modal"
+    >
+      <div class="fr-container fr-container--fluid fr-container-md">
+        <div class="fr-grid-row fr-grid-row--center">
+          <div class="fr-col-12 fr-col-md-8">
+            <div class="fr-modal__body">
+              <div class="fr-modal__header">
+                <button
+                  class="fr-btn--close fr-btn"
+                  :title="t('Close the modal dialog')"
+                  :aria-controls="id"
+                >
+                  {{ t("Close") }}
+                </button>
+              </div>
+              <form
+                @submit.prevent="submit"
+              >
+                <div class="fr-modal__content">
+                  <h1
+                    :id="modalTitleId"
+                    class="fr-modal__title fr-mb-2w"
+                  >
+                    {{ t("Add member to the organization") }}
+                  </h1>
+                  <InputGroup
+                    :label="t('Search a user')"
+                    :model-value="userId"
+                    @update:model-value="(user) => userId = user"
+                  />
+                  <SelectGroup
+                    :label="t('Role of the member')"
+                    v-if="roles.length > 0"
+                    :model-value="role"
+                    @update:model-value="(newRole) => role = (newRole as MemberRole)"
+                    :options="roles"
+                  />
+                </div>
+                <div class="fr-modal__footer">
+                  <div class="fr-btns-group fr-btns-group--right fr-btns-group--sm fr-btns-group--inline-lg fr-btns-group--icon-left">
+                    <button
+                      class="fr-btn fr-btn--secondary fr-btn--secondary-grey-500"
+                      @click="closeModal"
```

Comment:
The form gets wrongly submitted if clicking on Cancel button if some value has been set in input
```suggestion
                      type="button"
                      @click="closeModal"
```

Reply:
🙏🏻